src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -537,9 +537,9 @@ void GeotaggedImagesPlugin::_handle_camera_info(const mavlink_message_t *pMsg, s
 {
     gzdbg << "Send camera info" << endl;
     _send_cmd_ack(pMsg->sysid, pMsg->compid, MAV_CMD_REQUEST_CAMERA_INFORMATION, MAV_RESULT_ACCEPTED, srcaddr);
-    static const char* vendor = "PX4.io";
-    static const char* model  = "Gazebo";
-    char uri[128] = {};
+    static const char vendor[32] = "PX4.io";
```

Comment:
If you're going to do this you should get the size from the message itself rather than have a magic number.

Reply:
Or ignore all of that and use string literals directly below.

Reply:
for some reason I didn't have access to sizeof(mavlink_camera_information_t.vendor_name).
there was magic number before so I thought It will be ok, but got you, I will figure that out.